torch/_inductor/codegen/mps.py

```diff
@@ -194,6 +194,9 @@ def floordiv(a: CSEVariable, b: CSEVariable) -> str:
     def floor(x: CSEVariable) -> str:
         return f"metal::floor({x})"
 
+    @staticmethod
+    def sign(x: CSEVariable) -> str:
+        return f"metal::sign({x})"
```

Comment:
Didn't know such function exists.. But if passes CI I guess...

Reply:
It is sometimes used to compute the loss. e.g.
https://github.com/pytorch/pytorch/issues/74439